frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.tsx

```diff
@@ -149,33 +166,54 @@ export function QueryColumnPicker({
       color,
       checkIsColumnSelected,
       handleSelect,
+      withInfoIcons,
     ],
   );
 
+  const renderItemExtra = useCallback(
+    item => (
```

Comment:
Design comment:

Up until now **selected** and **hovered** items had the same styling, i.e.
- background was blue
- the temporal bucket picker has been shown

To match the above, the info icon should be always shown for the **selected** item. But it's shown only on hover.

cc: @mazameli

Reply:
This applies to all popovers (not just the one added in this PR), I'll add it to the product doc and we can review it during the design review.

Reply:
As per @mazameli's [comments on Slack](https://metaboat.slack.com/archives/C0645JP1W81/p1709136662459169?thread_ts=1708701338.803019&cid=C0645JP1W81), the current implementation is what we want to go with